### DISCUSSION TITLE
Welcome to Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI Discussions!

### DISCUSSION THREAD

**Discussion by diStyApps** [diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by recycleaway7** [comment]
Can you add some info on how to run this?

**Comment by diStyApps** [comment]
ill do that, but for now,

To start the app on a Windows machine, follow these steps:

Go to the [releases](https://github.com/diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI/releases) page 
Download the latest release of "Safe-and-Stable-Ckpt2Safetensors-GUI"
Unzip the file
Navigate to the folder where the file was extracted
Click on "run_app_gui.exe" to launch the app.

**Reply by recycleaway7** [reply]
> ill do that, but for now,
> 
> To start the app on a Windows machine, follow these steps:
> 
> Go to the [releases](https://github.com/diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI/releases) page Download the latest release of "Safe-and-Stable-Ckpt2Safetensors-GUI" Unzip the file Navigate to the folder where the file was extracted Click on "run_app_gui.exe" to launch the app.

Thanks! Not sure what I did wrong, but I initially installed it with a git clone command line and only ended up with a fraction of the files. I didn't have the run_app_gui.exe file or I would have been able to figure out that was the launch file. Getting the full zip file on the releases page worked.
